### PR TITLE
Mccalluc/adapt angular tut

### PR DESCRIPTION
**Pull request by mccalluc**
@thinkh: Here's the work I had in mind with the Angular tutorial. Ideally, the code that is displayed would be the same code that runs... but the fiddlejs is also nice, so leaving that for the time being.

### CODE REVIEW

**Comment by thinkh** [comment]
The automatic update from Travis looks great! :+1: 

Regarding the code example: I agree that it would be nice if the code of the running example would be the same as the one in the tutorial. The Gist usage had the advantage that people can easily fork the Gist and run the [Gist in JSFiddle](http://doc.jsfiddle.net/use/gist_read.html). (We [researched also a few alternatives](https://docs.google.com/document/d/1Q0wbQKj7HXlvg-CzLmU0Hwt07r0mZm7kTOMIMDJkxWI/edit#heading=h.p08lrw4xsu0r).)
I suggest to add an index.html (or similar) with the working demo. Then we can replace the inline JSFiddle and link to the running demo. What do you think @mccalluc?

**Comment by mccalluc** [comment]
Trying to make the whole example work locally, but I'm getting errors from angular:
```
Error: [$injector:modulerr] http://errors.angularjs.org/1.5.0/$injector/modulerr?p0=matrixApp&p1=%5B%24injector%3Anomod%5D%20http%3A%2F%2Ferrors.angularjs.org%2F1.5.0%2F%24injector%2Fnomod%3Fp0%3DmatrixApp%0AH%2F%3C%40http%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.5.0%2Fangular.min.js%3A6%3A416%0Ake%2F%3C%2F%3C%2F%3C%40http%3A%2F
```
@thinkh: If this is the approach you had in mind, do you have thoughts on fixing this error?

(Travis failure is unrelated: docu_nav.yml needs to change.)

**Comment by mccalluc** [comment]
I've just merged in a much simpler Angular tutorial, so I'm going to close this for the sake of housekeeping. In the future, a "Angular Integration, Part II" wouldn't be bad.
